src/scripts/main.js

```diff
@@ -354,7 +354,42 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+function addAgeCenturi(array) {
+  array.map((person) => {
```

Comment:
map always returns a new array, if you don't need that, then better to use forEach or rewrite your logic for map